### PR TITLE
Add Adobe Stock

### LINKED ISSUE #5088: Request: Adobe Stock Icon
Brand Name: Adobe Stock
Website: https://stock.adobe.com/
Alexa rank: 40

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
adobestock

**Issue:** closes #5088 

**Similarweb rank:**
The similarweb rank is ¿unavailable?. See https://www.similarweb.com/website/stock.adobe.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website header which was provided as SVG https://stock.adobe.com/
Color: `#0C131F` Taken from SVG in website header

### CODE REVIEW

**Review by LitoMore — CHANGES_REQUESTED** [review]
image

@PratyushJayachandran I used the logo on their homepage as a reference. But I found some differences. Could you check this?

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran I used the logo on their homepage as a reference. But I found some differences. Could you check this?

Should be fixed now.

**Comment by PratyushJayachandran** [comment]
> Thanks for this @PratyushJayachandran! I can see some minor differences around the text of yours (black) against the source file in the header (red) - would you mind taking a look?
 
I am only resizing to 24px width and subtracting St from the base. No other scaling or correction was done. 
I redid the process again and pushed a commit.

By the way I am using [Download All Images](https://addons.mozilla.org/en-US/firefox/addon/save-all-images-webextension/) extension on Firefox to pull the SVG, similar to what I do for other icon additions. I try to pull the website code itself when I can but in this case it leads to a broken image for some reason.

**Comment by PratyushJayachandran** [comment]
> Thanks for this @PratyushJayachandran! I can see some minor differences around the text of yours (black) against the source file in the header (red) - would you mind taking a look?

I have fixed it. I see no difference on my end.
image

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/753b84db-53d7-4d3f-aa7d-f93ca84a43a0) when comparing the source (red and green) to your PR (black). Could you try using a precision  of 4 for this icon?

**Comment by PratyushJayachandran** [comment]
> I'm still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/753b84db-53d7-4d3f-aa7d-f93ca84a43a0) when comparing the source (red and green) to your PR (black). Could you try using a precision of 4 for this icon?

I have increased precision and checked on my end. Please review again. Sorry for the multiple reviews.

**Comment by adamrusted** [comment]
> Sorry for the multiple reviews.

No stress! That's what we're here for! Let me have a look.

**Comment by adamrusted** [comment]
@PratyushJayachandran - you're going to hate me...
image

@LitoMore - what is our margin of error on these things? This one icon seems to be acting up a LOT.

**Comment by LitoMore** [comment]
Yes, I can also see the difference on my side. But it's acceptable since their control point positions are precisely the same. This may be a relatively unavoidable calculation error.

image

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - you're going to hate me... image

image
Should be fixed now. 🤞

**Review by adamrusted — COMMENTED** [review]
Another example of me losing my mind, I'm still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/508733bb-ffa3-454f-be70-ab605f7faab0) when comparing to the header SVG. @LitoMore - please feel free to veto me on this one if it's correct for you.

**Comment by PratyushJayachandran** [comment]
> Another example of me losing my mind, I'm still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/508733bb-ffa3-454f-be70-ab605f7faab0) when comparing to the header SVG. @LitoMore - please feel free to veto me on this one if it's correct for you.

I'm wondering how I'm missing it.
I used to check with the SVG before I optimize, which is wrong.
Now I open the SVG from icons/ and then paste original scaled to 24px on the bottom. To check for overlapped portions, I switch to paths only and zoom in to see any differences. Is there a better way? Am I missing something? 🤔

**Comment by PratyushJayachandran** [comment]
> All I do is take the source icon from the website, scale down to fit 24*24 viewport, and copy-paste that content into the PR version, which is why I think I'm just going a bit mad!

Can you check again. I checked on my side. If you still see a difference, provide me the source svg you are comparing against. Might be that we are looking at different sources.

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - I'm still seeing [differences](https://github.com/simple-icons/simple-icons/assets/1830867/198f3171-490e-49e6-a737-2608d528b5d5) when compared to the SVG ripped directly from the website header, see SVG code below.
> 
> ```
> <svg viewBox="0 0 700 149.5" width="108"><g><path fill="#0C131F" d="M0 122V27.4c0-1.8.1-3.6.5-5.3s.9-3.5 1.6-5.1 1.5-3.2 2.5-4.8S6.8 9.3 8 8c1.3-1.3 2.7-2.4 4.2-3.4s3.1-1.9 4.8-2.5c1.7-.7 3.4-1.2 5.1-1.6 1.8-.4 3.5-.5 5.3-.5h100.2c1.8 0 3.6.1 5.3.5 1.8.4 3.5.9 5.1 1.6s3.2 1.5 4.8 2.5 2.9 2.2 4.2 3.4c1.3 1.3 2.4 2.7 3.4 4.2s1.9 3.1 2.5 4.8c.7 1.7 1.2 3.4 1.6 5.1.4 1.8.5 3.5.5 5.3V122c0 1.8-.1 3.6-.5 5.3-.4 1.8-.9 3.5-1.6 5.1-.7 1.7-1.5 3.2-2.5 4.8s-2.2 2.9-3.4 4.2c-1.3 1.3-2.7 2.4-4.2 3.4s-3.1 1.9-4.8 2.5c-1.7.7-3.4 1.2-5.1 1.6-1.8.4-3.5.5-5.3.5H27.5c-1.8 0-3.6-.1-5.3-.5-1.8-.4-3.5-.9-5.1-1.6-1.7-.7-3.2-1.5-4.8-2.5s-2.9-2.2-4.2-3.4c-1.3-1.3-2.4-2.7-3.4-4.2s-1.9-3.1-2.5-4.8c-.7-1.7-1.2-3.4-1.6-5.1-.4-1.6-.6-3.4-.6-5.3z"></path><path fill="#FFFFFF" d="M54.9 109.6c-3.8 0-7.3-.4-10.4-1-3.1-.6-5.7-1.5-7.9-2.5-.6-.3-.8-.9-.8-1.7V92.2c0-.3.1-.4.3-.6.1-.1.4-.1.6.1 2.9 1.8 5.9 3.1 9.1 4 3.2.9 6.3 1.3 9.4 1.3 3.9 0 6.7-.6 8.4-1.8 1.7-1.2 2.5-2.7 2.5-4.5 0-1.2-.3-2.2-.8-3.1-.6-.9-1.5-1.8-2.9-2.7-1.4-.9-3.2-1.8-5.6-2.7L51.7 80c-4.2-1.8-7.4-3.7-9.8-5.7-2.4-2-4-4.2-4.8-6.5-.9-2.3-1.4-4.8-1.4-7.5 0-3.7.9-7.1 2.7-10 1.9-3 4.5-5.4 8.2-7.1 3.6-1.7 8.2-2.6 13.6-2.6 3.2 0 6.3.2 9.2.6 2.8.4 5.1 1.1 6.8 1.9.5.3.7.7.7 1.3V56c0 .1-.1.3-.3.4-.1.1-.4.1-.6-.1-2-1.1-4.5-1.9-7.1-2.6-2.7-.4-5.6-.7-8.5-.7-1.9 0-3.7.1-5 .5-1.4.4-2.5.8-3.3 1.3s-1.5 1.2-1.9 1.9c-.4.8-.6 1.6-.6 2.4 0 1.1.3 2.1.9 2.9.6.9 1.7 1.7 3.1 2.6 1.4.9 3.4 1.9 5.9 3l3.8 1.4c4.5 1.9 8 3.8 10.5 5.9 2.5 2.1 4.3 4.3 5.3 6.8s1.5 5 1.5 7.9c0 4-1.1 7.6-3.2 10.6-2.2 3-5.1 5.4-8.9 7s-8.4 2.4-13.6 2.4zm60.3-11.1v8.4c0 .7-.3 1.2-.8 1.3-1.3.4-2.7.7-4.1 1s-3.1.4-4.9.4c-4.5 0-7.9-1.2-10.5-3.5-2.5-2.3-3.8-6.1-3.8-11.2V69.6H85c-.6 0-.8-.3-.8-.9V58.5c0-.6.3-.8.9-.8h6.1c.1-1.2.1-2.5.3-4 .1-1.5.2-3.1.4-4.6.1-1.5.3-2.8.4-3.8.1-.2.1-.4.3-.6.1-.1.3-.3.5-.4l12.2-1.5c.2-.1.4-.1.5 0 .1.1.2.3.2.6-.1 1.6-.2 3.7-.3 6.4-.1 2.7-.1 5.3-.2 7.9h9.5c.4 0 .6.3.6.8v10.4c0 .4-.1.6-.5.7h-9.7v21.9c0 2.3.4 4 1.2 5 .8 1 2.2 1.5 4.3 1.5.6 0 1.2 0 1.7-.1.6 0 1.1-.1 1.7-.1.2-.1.4-.1.6.1.2.1.3.3.3.6z"></path><path fill="#0C131F" d="M213.2 79.6h15.1c-.3-1.3-.8-2.7-1.4-4.3-.6-1.6-1-3.3-1.6-5.1s-1.1-3.6-1.7-5.5c-.6-1.8-1.1-3.6-1.6-5.2-.6-1.7-.9-3.2-1.3-4.6h-.1c-.6 2.2-1.3 4.7-2 7.5-.9 3-1.7 5.9-2.7 8.9-.9 2.9-1.8 5.7-2.7 8.3zM232 91.5h-22.2l-4.7 14.2c-.1.6-.6.8-1.1.8h-11c-.7 0-.9-.3-.8-.9l19.2-55.4c.2-.6.3-1.1.6-1.9.2-.8.3-2 .3-4 0-.5.2-.7.6-.7h15.4c.5 0 .7.2.8.6l21.8 61.6c.2.6 0 .9-.5.9h-12.5c-.6 0-.9-.2-1-.7L232 91.5zm51.9 4v-26c-.5-.2-1-.3-1.6-.3-.7-.1-1.3-.1-1.9-.1-2.4 0-4.7.5-6.8 1.5-2 1-3.8 2.5-5 4.4s-1.9 4.4-1.9 7.5c0 2.4.3 4.4.9 6.1.7 1.7 1.6 3.2 2.7 4.2 1.1 1.1 2.5 1.9 4 2.5 1.6.6 3.2.8 5.1.8.8 0 1.6 0 2.4-.1.7 0 1.4-.3 2.1-.5zm-6 12.1c-4.8 0-9-.9-12.9-2.7-3.9-1.8-6.8-4.6-9-8.1-2.2-3.6-3.3-8.2-3.3-13.7 0-4.7 1.1-8.9 3.3-12.6 2.2-3.9 5.3-6.8 9.6-9.2 4.2-2.3 9.2-3.4 15-3.4h1.4c.5 0 1 .1 1.8.1V39.1c0-.5.2-.7.7-.7h12.1c.3 0 .6.2.6.6v56.8c0 1.1 0 2.3.1 3.6.1 1.4.1 2.5.3 3.4 0 .5-.2.7-.7.9-3.3 1.4-6.6 2.4-9.8 3-3.1.5-6.2.9-9.2.9zM328.2 69c-1.8 0-3.5.6-5 1.6s-2.6 2.6-3.5 4.7c-.9 2-1.3 4.4-1.3 7.4 0 2.8.5 5.2 1.3 7.3.8 2 2 3.6 3.6 4.8s3.4 1.7 5.6 1.7c1.8 0 3.5-.6 5-1.6s2.7-2.6 3.5-4.7 1.3-4.6 1.3-7.5c0-2.6-.3-4.9-1-6.9s-1.8-3.8-3.3-5c-1.6-1.2-3.7-1.8-6.2-1.8zm.6-11.1c3.9 0 7.3.7 10.2 2 2.8 1.4 5.3 3.2 7.3 5.5 1.9 2.3 3.5 4.9 4.4 7.9 1 3 1.5 6 1.5 9.3 0 5-1 9.3-3.1 13.1-2 3.8-4.8 6.7-8.3 8.8-3.5 2.2-7.6 3.2-12.3 3.2-4.1 0-7.6-.7-10.6-2.2-3-1.4-5.5-3.3-7.4-5.7-1.9-2.4-3.4-5-4.3-8-.9-3-1.5-5.9-1.5-9.1 0-4.6 1-8.6 3-12.4 1.9-3.8 4.7-6.7 8.3-9 3.7-2.3 7.9-3.4 12.8-3.4zm62 24.1c0-2.8-.6-5.2-1.6-7.2-1-1.9-2.4-3.3-4.2-4.2-1.8-.9-3.9-1.4-6-1.4-1.6 0-2.7.1-3.5.2-.8.1-1.5.3-2.2.6v26.1c.6.1 1.1.2 1.8.3.6.1 1.3.1 1.8.1 2.5 0 4.8-.6 6.8-1.6s3.8-2.6 5.1-4.7c1.3-2.2 2-4.9 2-8.2zm-30.2-43.6h11.7c.7 0 .9.2.9.8v19.7c1.3-.3 2.6-.6 4-.8 1.4-.2 2.7-.3 4.1-.3 4.8 0 8.9 1 12.3 3.1 3.4 2 6 4.8 8 8.2 1.9 3.4 2.8 7.3 2.8 11.6 0 4.4-.8 8.4-2.4 11.7-1.6 3.4-3.8 6.1-6.5 8.3-2.7 2.2-5.8 3.9-9.3 5-3.4 1.1-6.9 1.7-10.6 1.7-2.6 0-5.1-.2-7.6-.6-2.5-.3-5-.9-7.4-1.7-.3-.1-.6-.5-.6-.9V39.1c0-.5.2-.7.6-.7zm62.7 38.5h16.1c.7 0 1.3-.1 1.4-.1v-.6c0-.6-.1-1.3-.3-2.2-.2-.8-.7-1.7-1.4-2.5-.7-.9-1.5-1.6-2.5-2.2-1-.6-2.4-.8-4-.8-1.9 0-3.5.5-4.9 1.3-1.4.9-2.3 1.9-3 3.2-.7 1.2-1.1 2.5-1.4 3.9zm18.9 9.3h-18.9c.2 1.8.8 3.5 1.8 5s2.5 2.7 4.4 3.6c1.9.9 4.6 1.4 7.6 1.4 2.2 0 4.2-.2 6.1-.6 1.9-.3 3.9-.9 5.7-1.7.3-.2.6-.1.6.5v9.1c0 .3 0 .6-.1.7-.1.1-.2.3-.6.5-1.8.9-4 1.6-6.3 2-2.3.5-5.1.7-8.4.7-4.6 0-8.3-.7-11.5-2.2-3.1-1.4-5.6-3.3-7.5-5.6-1.9-2.4-3.3-5-4.2-7.9-.9-2.8-1.3-5.8-1.3-8.8 0-3.3.6-6.5 1.6-9.6 1-3.1 2.5-5.8 4.4-8.1 1.9-2.4 4.3-4.2 7.3-5.6 2.8-1.4 6.3-2 10-2 3.6 0 6.7.6 9.3 1.8 2.6 1.3 4.7 2.8 6.4 5 1.6 2 2.8 4.3 3.5 6.7.7 2.4 1.1 4.9 1.1 7.2 0 1.5 0 2.7-.1 4-.1 1.3-.2 1.9-.3 2.5-.1.5-.3.7-.8.8-.5.1-1.1.1-2 .2-.9.1-2 .1-3.4.1-1.2.3-2.7.3-4.4.3zm55 21.4c-3.5 0-6.7-.3-9.7-.9-3-.6-5.3-1.5-7.4-2.5-.5-.2-.7-.8-.7-1.6V91.1c0-.2.1-.5.2-.6.1-.1.3-.1.6 0 2.7 1.7 5.6 3 8.5 3.8 3 .8 5.9 1.3 8.9 1.3 3.6 0 6.3-.6 7.9-1.7s2.4-2.5 2.4-4.2c0-1-.2-1.9-.8-2.8s-1.5-1.7-2.7-2.6c-1.3-.9-3-1.7-5.2-2.6l-4.8-2c-3.9-1.7-6.9-3.4-9.2-5.3-2.2-1.9-3.8-3.9-4.7-6.1-.9-2.2-1.3-4.6-1.3-7.1 0-3.4.9-6.6 2.6-9.4 1.7-2.8 4.3-5 7.7-6.7s7.6-2.5 12.7-2.5c3 0 5.8.2 8.5.6 2.7.3 4.8 1 6.5 1.9.3.2.6.7.6 1.3v10.8c0 .2-.1.3-.2.6-.1.1-.3.1-.6 0-1.9-1.1-4.2-1.9-6.7-2.4-2.6-.6-5.2-.8-8.1-.8-1.8 0-3.3.1-4.7.5-1.3.3-2.3.7-3.1 1.3-.8.6-1.4 1.1-1.7 1.8s-.6 1.5-.6 2.2c0 1 .2 1.9.8 2.7.6.8 1.5 1.6 2.8 2.5 1.4.8 3.2 1.7 5.6 2.7l3.5 1.5c4.3 1.7 7.6 3.6 9.9 5.6 2.4 1.9 4 4.1 5 6.4.9 2.3 1.5 4.7 1.5 7.2 0 3.9-1 7.2-3 10s-4.8 5-8.3 6.6c-3.4 1.2-7.7 2-12.7 2zM552.8 97v8c0 .7-.2 1.1-.7 1.3-1.4.5-2.7.8-4 1-1.4.2-2.8.3-4.6.3-4.2 0-7.5-1.1-9.8-3.3-2.3-2.2-3.4-5.7-3.4-10.5V69.9h-5.8c-.6-.1-.8-.3-.8-.9v-9.4c0-.6.3-.9.9-.9h5.8c.1-1.1.1-2.4.2-3.9s.1-3 .3-4.3c.1-1.4.3-2.6.5-3.4.1-.2.1-.5.3-.6.1-.1.3-.2.6-.3l11.3-1.5c.2-.1.5-.1.6 0 .1.1.2.2.2.6-.1 1.5-.2 3.5-.3 6-.1 2.5-.2 5-.2 7.4h9c.6 0 .8.2.8.8v9.7c0 .3-.2.6-.7.7h-9v20.6c0 2.2.3 3.6 1.1 4.7.7 1 2 1.5 4.1 1.5.6 0 1.1 0 1.6-.1s1-.1 1.6-.3c.2 0 .3 0 .5.1.1-.1-.1.3-.1.6zm27.3-28c-1.8 0-3.5.6-5 1.6s-2.6 2.6-3.5 4.7c-.9 2-1.3 4.4-1.3 7.4 0 2.8.5 5.2 1.3 7.3s2 3.6 3.6 4.8 3.4 1.7 5.6 1.7c1.8 0 3.5-.6 5-1.6s2.7-2.6 3.5-4.7c.8-2 1.3-4.6 1.3-7.5 0-2.6-.3-4.9-1-6.9s-1.8-3.8-3.3-5c-1.6-1.2-3.7-1.8-6.2-1.8zm.6-11.1c3.9 0 7.3.7 10.2 2 2.8 1.4 5.3 3.2 7.3 5.5 1.9 2.3 3.5 4.9 4.4 7.9 1 3 1.5 6 1.5 9.3 0 5-1 9.3-3.1 13.1-2 3.8-4.8 6.7-8.3 8.8-3.5 2.2-7.6 3.2-12.3 3.2-4.1 0-7.6-.7-10.6-2.2-3-1.4-5.5-3.3-7.4-5.7-1.9-2.4-3.4-5-4.3-8-.9-3-1.5-5.9-1.5-9.1 0-4.6 1-8.6 3-12.4s4.7-6.7 8.3-9c3.7-2.3 7.9-3.4 12.8-3.4zm65.4 37.3v10c0 .3-.1.7-.5.8-1.6.7-3.2 1-5.1 1.3-1.8.2-3.8.3-5.7.3-3.9 0-7.4-.6-10.5-1.8-3.1-1.3-5.8-3-8-5.2s-3.9-4.9-5.1-7.9c-1.1-3-1.8-6.3-1.8-9.8 0-4.8 1.1-9.1 3.4-12.9 2.3-3.8 5.3-6.7 9.4-9 4.1-2.2 8.9-3.3 14.3-3.3 2.4 0 4.2.1 5.6.3 1.4.2 2.5.5 3.2.8.3.1.6.5.6 1l-.1 9.9c0 .6-.2.8-.8.7-.9-.3-2-.7-3.4-.9-1.4-.2-3-.3-4.7-.3-2.7 0-5.1.5-7.2 1.5-2 .9-3.8 2.4-5 4.3-1.3 1.9-1.8 4.3-1.8 7.3 0 3.3.7 5.9 2.2 7.9 1.4 1.9 3.2 3.3 5.5 4.2 2.2.8 4.4 1.3 6.6 1.3 1.7 0 3.3-.1 4.7-.2 1.4-.1 2.5-.3 3.3-.6.6-.3.9-.2.9.3zm21.1-55.9v66.3c0 .7-.3.9-.9.9h-11.6c-.7 0-.9-.3-.9-.9V39.2c0-.2.1-.3.2-.6.1-.1.3-.2.8-.2h11.5c.5 0 .9.3.9.9zm17.3 19.5h12.4c.3 0 .6.1.7.3.1.2 0 .5-.1.7-.7.8-1.6 1.8-2.7 3.1-1.1 1.4-2.5 2.7-4 4.4-1.5 1.6-3 3.3-4.4 4.9-1.5 1.6-2.8 3.2-4.1 4.6-1.3 1.4-2.3 2.5-3 3.3-.1.1-.1.1-.1.2s0 .1.1.2c1.4 1.7 3 3.6 4.7 5.9 1.7 2.2 3.5 4.4 5.5 6.7s3.8 4.6 5.6 6.7c1.8 2.2 3.4 4.2 4.9 5.9.2.2.2.3.1.6-.1.2-.3.3-.6.3h-15.2c-.3 0-.6 0-.7-.1-.1-.1-.3-.2-.6-.6-.8-1.1-2-2.6-3.6-4.7-1.6-2-3.3-4.3-5.2-6.7-1.8-2.4-3.5-4.7-5.1-6.8-1.6-2.2-2.8-3.8-3.8-4.9-.2-.3-.5-.7-.5-.9-.1-.2.1-.6.5-.9.8-1 1.9-2.3 3.2-3.9 1.4-1.6 2.7-3.2 4.2-5s3-3.5 4.3-5.2c1.4-1.7 2.6-3.2 3.8-4.6s1.8-2.3 2.3-2.8c.2-.3.5-.6.6-.6.2 0 .4-.1.8-.1z"></path></g></svg>
> ```

I'll use the SVG you attached. This is an ongoing saga 😂

**Review by adamrusted — CHANGES_REQUESTED** [review]
@PratyushJayachandran - updating this, I'm still seeing issues around the letters as below.
image

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - updating this, I'm still seeing issues around the letters as below. image

Oof.. I give up. Somebody else take this. It could be because of differences in the software used. Funny that it's happening on Adobe Stock icon itself.

File: icons/adobestock.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...3038-1.114-.4051-.4219-.1012-.8607-.135-1.3164-.135-.3038 0-.5738.0338-.7764.0844-.2194.0506-.3881.1181-.5232.2025-.135.0844-.2363.1857-.287.3038-.0674.1181-.1012.2531-.1012.3713 0 .1688.0507.3207.135.4557.1013.135.2532.27.4726.405.2194.1351.5232.287.9283.4726l.5907.2195c.692.2869 1.249.5907 1.6371.9113.3882.3207.6751.6751.827 1.0465.152.3713.2363.7763.2363 1.2151 0 .6245-.1687 1.1646-.5063 1.6372zm5.9072-4.827c0 .0675-.0338.1012-.0844.118h-1.5021v3.3419c0 .3544.0675.6245.1856.7764.1182.1519.3545.2362.6751.2362h.27c.0845 0 .1689-.0168.2532-.0168.0338-.017.0676 0 .0844 0 .0507 0 .0675.0337.0675.0844v1.2995c0 .1182-.0337.1857-.135.2026a3.5697 3.5697 0 0 1-.6413.1519c-.2194.0506-.4726.0675-.7595.0675-.692 0-1.2321-.1857-1.6203-.54-.3882-.3545-.5907-.9284-.5907-1.7216v-3.9325h-.9283c-.0844 0-.135-.0506-.135-.135V9.5106c0-.0844.0506-.135.135-.135h.9283c.0169-.1857.0169-.3883.0338-.6246s.0337-.4725.0675-.7088c.0337-.2363.0506-.4388.0675-.5738.0168-.0338.0337-.0676.0506-.0844a.093.093 0 0 1 .0844-.0507l1.8903-.2363c.0337-.0168.0506-.0168.0844 0 .0169.017.0338.0338.0338.1013-.017.2363-.0338.5739-.0507.9958-.0169.422-.0169.827-.0338 1.2152h1.4684c.0675 0 .1013.0506.1013.135v1.6034z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/adobestock.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Adobe Stock</title><path d="M19.833.75H4.167A4.167 4.167 0 0 0 0 4.917v14.166a4.167 4.167 0 0 0 4.167 4.167h15.666A4.167 4.167 0 0 0 24 19.083V4.917A4.167 4.167 0 0 0 19.833.75Zm-7.375 15.21a3.24 3.24 0 0 1-1.409 1.103 5.361 5.361 0 0 1-2.126.387 8.027 8.027 0 0 1-1.635-.15 5.409 5.409 0 0 1-1.231-.395.276.276 0 0 1-.129-.274v-1.919a.096.096 0 0 1 .04-.088.108.108 0 0 1 .105.008 5.647 5.647 0 0 0 2.915.838 2.328 2.328 0 0 0 1.313-.283.836.836 0 0 0 .392-.7.942.942 0 0 0-.127-.49 1.403 1.403 0 0 0-.45-.428 5.341 5.341 0 0 0-.885-.435l-.806-.338a6 6 0 0 1-1.537-.9 2.572 2.572 0 0 1-.765-1.021 3.272 3.272 0 0 1-.208-1.176 2.961 2.961 0 0 1 .433-1.578 3 3 0 0 1 1.288-1.119A4.899 4.899 0 0 1 9.761 6.6a10.079 10.079 0 0 1 1.441.096 3.416 3.416 0 0 1 1.07.3.224.224 0 0 1 .113.208v1.824a.111.111 0 0 1-.04.074c-.027.027-.063.024-.105-.009a4.491 4.491 0 0 0-1.127-.402 6.033 6.033 0 0 0-1.336-.15 3.28 3.28 0 0 0-.789.081 1.978 1.978 0 0 0-.524.201.777.777 0 0 0-.3.3.804.804 0 0 0-.096.378.75.75 0 0 0 .15.459 1.572 1.572 0 0 0 .483.404 10.175 10.175 0 0 0 .933.473l.6.226a6.48 6.48 0 0 1 1.652.937 2.823 2.823 0 0 1 .837 1.062 3.141 3.141 0 0 1 .241 1.232 2.814 2.814 0 0 1-.507 1.666zm4.588-.579a.836.836 0 0 0 .684.233c.096 0 .188 0 .273-.008.086-.007.172-.013.258-.024a.1.1 0 0 1 .088.007c.027.017.041.052.041.106v1.32c0 .118-.043.187-.129.21a5.763 5.763 0 0 1-.643.16 4.242 4.242 0 0 1-.774.065 2.333 2.333 0 0 1-1.642-.548 2.279 2.279 0 0 1-.6-1.755v-4.014h-.967c-.087 0-.129-.048-.129-.15l.012-1.586c0-.085.048-.129.15-.129h.974c.01-.183.011-.382.027-.623.014-.242.037-.483.064-.725.027-.242.051-.436.072-.588a.3.3 0 0 1 .05-.088.174.174 0 0 1 .079-.056l1.915-.241a.106.106 0 0 1 .081 0c.023.01.033.041.033.095-.022.248-.037.583-.049 1.007-.012.425-.009.822-.02 1.22h1.41c.065 0 .098.043.098.128l-.012 1.617a.099.099 0 0 1-.081.113H16.89v3.466a1.288 1.288 0 0 0 .156.788z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
I remember this one! It's very much _not_ fun!

I found a single path, monochrome SVG at https://creativecloud.adobe.com/apps/all/desktop, which differs from the logo and still has too many points in the letters, but at least the corners of the square are cleaner. Might be worth running with it to get this one off our plate.

Here's the path from the original:
```
m15.222,2H4.778c-1.53425,0-2.778,1.24375-2.778,2.778v9.443c-.00055,1.53425,1.24275,2.77845,2.777,2.779h10.445c1.53425,0,2.778-1.24375,2.778-2.778V4.778c0-1.53425-1.24375-2.778-2.778-2.778Zm-4.922,10.14c-.23775.32833-.56318.58306-.939.735-.44996.18116-.93206.26894-1.417.258-.36546.00399-.73037-.02952-1.089-.1-.28329-.05417-.55897-.14248-.821-.263-.06245-.03804-.09627-.1096-.086-.182v-1.28c-.0026-.02315.00779-.04584.027-.059.02258-.01079.04918-.00889.07.005.29665.18463.61826.32574.955.419.32115.0911.65318.13819.987.14.30382.02485.60821-.04055.875-.188.16172-.10049.26091-.27661.263-.467.00161-.11479-.02812-.22785-.086-.327-.07739-.116-.17993-.21308-.3-.284-.18718-.11487-.38471-.21196-.59-.29l-.537-.225c-.36967-.14778-.71515-.35002-1.025-.6-.22464-.18084-.40004-.4154-.51-.682-.09408-.25023-.1412-.51567-.139-.783-.01531-.7766.43738-1.48639,1.148-1.8.44767-.19437.93331-.28562,1.421-.267.32145-.00167.64261.01972.961.064.24735.0271.48856.09467.714.2.0497.02801.07888.08208.075.139v1.216c-.00249.01888-.01216.03607-.027.048-.018.018-.041.016-.07-.005-.23725-.12245-.48983-.21259-.751-.268-.29238-.06623-.59121-.09977-.891-.1-.17686-.00313-.35346.015-.526.054-.12222.0275-.23978.07264-.349.134-.08399.04674-.15326.11601-.2.2-.04158.07752-.06355.16404-.064.252-.00058.11012.0345.21747.1.306.08844.10987.19791.20098.322.268.20211.11626.41005.22206.623.317l.4.15c.39528.15337.7659.36395,1.1.625.24052.1874.432.43034.558.708.10853.25999.1633.53927.161.821.00649.39735-.11364.78646-.343,1.111Zm3.059-.386c.12039.1186.28819.176.456.156.06667,0,.12767-.00167.183-.005.057,0,.114-.009.172-.016.0194-.00743.04113-.00559.059.005.018.011.027.034.027.07v.88c0,.079-.029.125-.086.14-.14081.04392-.28405.07964-.429.107-.16997.03024-.34237.04464-.515.043-.3983.02322-.79029-.10745-1.095-.365-.29299-.31576-.43836-.74095-.4-1.17v-2.677h-.641c-.05733,0-.086-.03333-.086-.1l.008-1.057c0-.057.032-.086.1-.086h.648c.007-.122.008-.255.019-.416s.025-.322.043-.483.034-.292.048-.392c.00706-.02142.0179-.0414.032-.059.01398-.01759.03272-.03078.054-.038l1.277-.161c.01728-.00722.03672-.00722.054,0,.014.007.021.029.021.064-.01467.16467-.02533.38833-.032.671-.007.283-.006.548-.013.813h.94c.043,0,.064.029.064.086l-.008,1.078c.00543.03553-.01858.06888-.054.075h-.946v2.311c-.01991.18071.0179.36309.108.521l-.008.005Z
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
@PeterShaggyNoble - using [this SVG](https://[auth.services.adobe.com/img/attributions/Stock.svg](https://auth.services.adobe.com/img/attributions/Stock.svg)) from the login page your URL presents, I'm seeing the below differences (your PR in black).

image

**Comment by adamrusted** [comment]
Closing as we're removing all Adobe brands at the end of the month. @PratyushJayachandran feel free to re-open a PR if/when you obtain permission.

**Pull request closed without merging**